js/pages/configuration/configuration.js

```diff
@@ -89,8 +91,16 @@ define([
       return this.sourceJobs.get(job.executionId);
     }
 
-    checkJobs() {
-      this.jobListing().forEach(job => {
+    async checkJobs() {
+      const notifications = await jobDetailsService.list([]);
```

Comment:
was: this.jobListing() - list of jobs from sharedState.jobListing.
now: jobDetailsService.list([]) which requests the same list only directly from backend.

It doesn't fix the problem. Either way, list doesn't contain "refresh jobs", started from configuration page by clicking on "refresh" button.